modules/core/include/opencv2/core/hal/intrin_rvv_scalable.hpp

```diff
@@ -475,6 +475,23 @@ OPENCV_HAL_IMPL_RVV_LUT(v_float32, float, m1)
 OPENCV_HAL_IMPL_RVV_LUT(v_float64, double, mf2)
 #endif
 
+inline v_uint32 v_lut(const unsigned* tab, const v_int32& vidx) \
+{ \
+    return vloxei32(tab, vreinterpret_u32m1(vidx), VTraits<v_uint32>::vlanes()); \
```

Comment:
Are these functions needed? GCC 13 compiler generates an error here, I tried to remove this block and it compiles just fine without it. I did not try clang yet.

Reply:
> Are these functions needed?

Ah, these functions is not needed in this patch. But they are used in others file, such as, `color_lab.cpp` line 86.

> GCC 13 compiler generates an error here, I tried to remove this block and it compiles just fine without it. 

Yes, I reproduced this error using GCC 13. The reason is lack of corresponding macros in the compatibility layer `modules/core/include/opencv2/core/hal/intrin_rvv_compat_overloaded.hpp`. After testing, adding `OPENCV_HAL_IMPL_RVV_FUN_LOXEI(vuint32m1_t, u32m1, vuint32m1_t, i32)` on line 47 will solve this problem

I will temporarily remove this code block. Then re-add it in the next patch, and fix related issues in the compatibility layer.